### DISCUSSION TITLE
The BTTV app won't let me sign in....

### DISCUSSION THREAD

**Discussion by HuntertheSquirrel** [bttv-android/bttv#593] [Q&A] [answered]
I try signing in to the Twitch on the BTTV app but it won't let me.

Is there an issue to fix?

**Comment by FoseFx** [comment, marked as answer]
The issue has been identified and fixed. It will take me a bit to publish this though, I hope I get it done by the weekend.

Related issue: https://github.com/bttv-android/bttv/issues/588
